modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -633,9 +438,7 @@ The following table lists the known issues and the version in which they were fi
 
 | Procedure signatures from `SHOW PROCEDURES` will render the vector arguments with a type of `ANY` rather than the semantically correct type of `LIST<INTEGER \| FLOAT>`.
 [NOTE]
-====
 The types are still enforced as `LIST<INTEGER \| FLOAT>`.
-====
 |
```

Comment:
I can see that this is left from before, but I think this table should only show fixed issues and when they were fixed, unfixed issues were up in the limitations list before

and I assume since this doesn't have a `Fixed in` version that it is not yet fixed, so either it should move up or the sentence above the column should mention something about `and if fixed, the version in which they were fixed:` instead of just `and the version in which they were fixed:` if not everything in the table has been fixed